docs/src/main/sphinx/installation/cli.rst

```diff
@@ -48,15 +48,59 @@ recognizes these certificates.
 
 Authentication
 --------------
+Password/LDAP Authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 You can override your username with the ``--user`` option. It defaults to your
 operating system username. If your Trino server requires password
 authentication, use the ``--password`` option to have the CLI prompt for a
 password. You can set the ``TRINO_PASSWORD`` environment variable with the
 password value to avoid the prompt.
 
+When Trino is configured with LDAP authentication, invoking CLI requires additional options
+such as ``--keystore-*`` or ``--truststore-*`` to secure TLS connection.
+The simplest way to invoke the CLI is with a wrapper script.
+
+.. code-block:: text
+
+    #!/bin/bash
+
+    ./trino \
+    --server https://trino-coordinator.example.com:8443 \
+    --keystore-path /tmp/trino.jks \
+    --keystore-password password \
+    --truststore-path /tmp/trino_truststore.jks \
+    --truststore-password password \
+    --catalog <catalog> \
+    --schema <schema> \
+    --user <LDAP user> \
+    --password
+
+=============================== =========================================================================
```

Comment:
All these parameters should be documented but probably in separate sections .. for the password/ldap section we only need --user and --password

--server should already be documented .. if not we need to add that in a separate PR as well ;-)

Reply:
Turns out server is not explicitly documented .. so we should add that separately or in this PR.